tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -605,16 +605,47 @@ TEST(caffe2, FC) {
     updateInputPlaceholdersByName(ctx, &mod, {"inputs"}, {&inputs});
   }
 
-  EE.compile(CompilationMode::Infer, F);
-  EE.run(ctx);
+  // High level check on the content of the graph. We have 1 FC node and 1 save.
+  EXPECT_EQ(F->getNodes().size(), 2);
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *fcNode =
+      llvm::dyn_cast<FullyConnectedNode>(saveNode->getInput().getNode());
+  ASSERT_TRUE(fcNode);
 
-  auto result = ctx.get(output)->getHandle();
-  std::vector<size_t> expectedDims = {2, 4};
-  std::vector<float> expectedValues = {14.1f, 32.2f, 50.3f,  68.4f,
-                                       32.1f, 77.2f, 122.3f, 167.4f};
-  EXPECT_TRUE(result.dims().vec() == expectedDims);
-  for (size_t i = 0; i < 2 * 4; i++)
-    EXPECT_FLOAT_EQ(result.raw(i), expectedValues[i]);
+  // Check the numerical values of the weights and biases.
+  {
+    const Constant *constant = mod.getConstantByName("weights");
+    ASSERT_TRUE(constant);
+    const Tensor &weights = constant->getPayload();
+    const std::vector<size_t> expectedDimensions = {3, 4};
+    const std::vector<float> expectedValues = {1.0f, 4.0f, 7.0f, 10.0f, //
+                                               2.0f, 5.0f, 8.0f, 11.0f, //
+                                               3.0f, 6.0f, 9.0f, 12.0f};
+    EXPECT_EQ(expectedDimensions, weights.dims().vec());
+    ASSERT_EQ(expectedValues.size(), weights.size());
+    const auto &elements = weights.getHandle();
```

Comment:
I don't think this is a reference type? `Constant::getHandle` returns a `Handle<ElemTy>`.

Reply:
You're right, it's not. Removed `&`.